pubspec.lock

```diff
@@ -106,6 +106,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "3.0.1"
+  dropdown_plus:
+    dependency: "direct main"
+    description:
+      name: dropdown_plus
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "0.0.9"
```

Comment:
I've added this since it was added to `pubspec.yaml`. But we should check if we need this because as I said, it doesn't play well with dark mode.